CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Next release
 
+- console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) #5706
+
```

Comment:
It doesn't require changelog.
```suggestion
```

Reply:
I think a changelog is good to have because this may affect users (those using inherited tables)

```
CREATE TABLE cities (
  name       text,
  population real,
  elevation  int     -- (in ft)
);

CREATE TABLE capitals (
  state      char(2)
) INHERITS (cities);

postgres=# alter table only cities drop column name;
ALTER TABLE
postgres=# select * from cities;
 population | elevation 
------------+-----------
(0 rows)

postgres=# select * from capitals;
 name | population | elevation | state 
------+------------+-----------+-------
(0 rows)
```

What has essentially happened above is that the set of inherited columns has reduced to the 2 columns: `population` and `elevation`. So the `capitals` table is still inherited from `cities` table;


The reverse is not possible though, you can't drop a column ONLY from the inherited table as it would not be compatible with it's parent table:

```
postgres=# alter table only capitals drop column population;
ERROR:  cannot drop inherited column "population"
```